app/controllers/events_controller.rb

```diff
@@ -13,9 +13,9 @@ def authenticate_event_admin!
   # GET /events
   # GET /events.json
   def index
-    @core_member = false
-    if !current_user.nil? and current_user.is_core_member
-      @core_member = true
+    @event_admin = false
```

Comment:
Talked with @aganpule offline about this. Will cause some controller tests to fail. fixing